jkamz/Questioner
# #162983098 Meetup details and questions

### What does this PR do?
This PR creates the html page for individual meetups

### Description of the tasks to be completed
The tasks for this PR include:

- Create a questions section
- Create an add question link

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-meetup-page-162983098`

Navigate to UI/templates folder and open meetup.html on browser

### What are the relevant Pivotal Tracker stories?
#162983098

### Screenshots

![meetup](https://user-images.githubusercontent.com/11965186/50742514-50c5c280-121d-11e9-8232-d0cedfa896d7.png)

Mobile Site

![meetupmobile](https://user-images.githubusercontent.com/11965186/50742517-5cb18480-121d-11e9-8764-ca1b6fe18726.png)
